ui/src/components/AppHeader.js

```diff
@@ -1,56 +1,69 @@
 import React from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 
-import AppBar from '@material-ui/core/AppBar';
 import IconButton from '@material-ui/core/IconButton';
-import Toolbar from '@material-ui/core/Toolbar';
-import Typography from '@material-ui/core/Typography';
 import MenuIcon from '@material-ui/icons/Menu';
 
 const useStyles = makeStyles(theme => ({
-  appBar: ({ drawerWidth }) => ({
+  appBar: {
     position: 'sticky',
-    ...(drawerWidth && {
-      [theme.breakpoints.up('sm')]: {
-        width: `calc(100% - ${drawerWidth}px)`,
-        marginLeft: drawerWidth,
-      },
-    }),
-  }),
+    [theme.breakpoints.up('sm')]: {
+      display: 'none',
+    },
+    padding: '8px',
```

Comment:
why 8px rather than 10 or 12 or 3? Any particular reason?

Reply:
8px increments for spacing are standard for material